libs/blocks/preflight/preflight.css

```diff
@@ -742,7 +742,7 @@ picture:has(.picture-meta) {
 picture:hover .picture-meta,
 img:hover ~ .picture-meta {
   opacity: 1;
-  z-index: 19;
+  z-index: 9;
```

Comment:
Gave it a [quick check](https://github.com/search?q=repo%3Aadobecom%2Fmilo%20z-index&type=code), seems like chart/list.css has a z-index of 10 for buttons? Ah our z-index strategy is a mess.
Further below in the file, we have a z-index of 103... should that be 9/10 as well?

Reply:
Can you also change the z-index for the performance LCP modal?